### PR TITLE
Added 6 new internships

### PR DESCRIPTION
**Pull request by ShevinuM**

### CODE REVIEW

**Comment by kanjurer** [comment]
The Autodesk one does not seem to be for Summer 2024.

**Comment by kanjurer** [comment]
Also for the Waabi one, https://jobs.lever.co/waabi/ad3c77da-5607-4c4f-8784-8407478cbace this might be a better link to put.

**Comment by ShevinuM** [comment]
> The Autodesk and Perpetua ones do not seem to be for Summer 2024. They are for Fall 2023, correct me if I am wrong
> 
> 

It is actually once you click and apply now. But I'll update the link to the workday one instead of that

**Comment by ShevinuM** [comment]
> Also for the Waabi one, https://jobs.lever.co/waabi/ad3c77da-5607-4c4f-8784-8407478cbace this might be a better link to put.
> 
> For Klue, https://jobs.lever.co/klue/b88e45c3-c067-410b-a7d3-ca23142b9bef link is better

I will make the changes

**Comment by ShevinuM** [comment]
> The Autodesk and Perpetua ones do not seem to be for Summer 2024. They are for Fall 2023, correct me if I am wrong

This was a mistake on my behalf. The internship starts early spans to Summer 24.

**Comment by ShevinuM** [comment]
Two of the internships have closed and I removed them. So there's still 2 active internships remaining which is in the list.

File: README.md [hunk 1]
```diff
@@ -19,8 +19,10 @@ To contribute:
...
-| Google                                                                                                                   | Toronto, Waterloo             | :lock: Closed :lock: | Software Developer Intern, BS, Summer 2024                      |
+| Ansys                                                                                                                    | Vancouver, Montreal, Waterloo | :lock: Closed :lock: | Software Development and Testing (Spring 2024)                 |
+| Google                                                                                                                   | Toronto, Waterloo             | :lock: Closed :lock: | Software Developer Intern, BS, Summer 2024                     |
+| [Waabi](https://jobs.lever.co/waabi/ad3c77da-5607-4c4f-8784-8407478cbace)                           | Toronto, ON, Canada           | Unknown              | Research Internship/Co-op                                   |
```
Comment (kanjurer) [inline README.md:24]: Nit: Indentation is a bit off. Could copy pasta this:

```
| [Waabi](https://jobs.lever.co/waabi/ad3c77da-5607-4c4f-8784-8407478cbace)                                                | Toronto, ON, Canada           | Unknown              | Research Internship/Co-op                                       |
| [Klue](https://jobs.lever.co/klue/b88e45c3-c067-410b-a7d3-ca23142b9bef)                                                  | Unknown                       | Unknown              | Data Analyst Intern                                             |
```

File: README.md [hunk 2]
```diff
@@ -19,8 +19,10 @@ To contribute:
...
+| [Klue](https://jobs.lever.co/klue/b88e45c3-c067-410b-a7d3-ca23142b9bef)                                  | Unknown                      | Unknown              | Data Analyst Intern                                             |
```
Comment (kanjurer) [inline README.md:25]: You can use prettier (or some sort of linter) in VS Code to do the linting for you. Makes it less troublesome 😓

**Review by kanjurer — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -19,8 +19,14 @@ To contribute:
 | [Konrad](https://boards.greenhouse.io/konradgroup/jobs/5654760003)                                                       | Toronto                       | Unknown              | Summer 2024 UofT PEY - Software Developer Intern (12-16 months) |
 | [Cisco](https://jobs.cisco.com/jobs/ProjectDetail/Software-Engineer-Intern-Canada/1405428?uid=WdGd8tgOdAzFLoUI)          | Ottawa                        | Rolling Applications | Software Engineer (Intern) Canada                               |
 | [Intuit](https://jobs.intuit.com/search-jobs/Toronto%2C%20Canada/27595/4/6251999-6093943-6167865/43x70011/-79x4163/50/2) | Toronto                       | Unknown              | Software Developer Co-op - 8 Months                             |
```
[the remark this answers is not in the document]
Comment (ShevinuM) [reply README.md:21]: https://waabi.ai
https://klue.com/careers

You could visit the websites and checkout. They redirect you to the lever website which I why it's not the company website that's included.

**Comment by ShevinuM** [comment]
I've made the changes requested. Also there is no 'company' link for the posting because it's the company itself who directs you to that link.

**Review by jenndryden — APPROVED** [review]
this looks great!

**Pull request merged**